### DISCUSSION TITLE
Is minification applied to `parcel serve`

### DISCUSSION THREAD

**Discussion by avalanche1** [parcel-bundler/parcel#5886] [Q&A] [answered]
During build process of `parcel serve` there is a line in log
`Optimizing popup.[hash].js`
What does that mean? The output files are not minified.

**Comment by mischnic** [comment, marked as answer]
Optimizer plugins are executed, but they all check `options.shouldOptimize` (which is `false` in serve mode):

https://github.com/parcel-bundler/parcel/blob/77c29cbf680746d4fef6161c7702178ed9b92d92/packages/optimizers/terser/src/TerserOptimizer.js#L14-L16

But there is also an optimizer that doesn't minify but converts the format into a data url:

https://github.com/parcel-bundler/parcel/blob/v2/packages/optimizers/data-url/src/DataURLOptimizer.js

So optimizers run, but no minification
